src/ui.cpp

```diff
@@ -87,7 +87,13 @@ extern void ui::draw(vbrush::iface* b, const size_t flags, const app_data& ad, c
 	if(flags & draw_flags::SHOW_MONSTER_DATA) {
 		b->next_row(2);
 		// then Monsters - first header
-		std::snprintf(buf, 256, "%-32s%-14s%-8s", "Monster Name", "HP", "%");
+		if(flags & draw_flags::SHOW_CROWN_DATA) {
+			std::snprintf(buf, 256, "%-32s%-14s%-8s %-6s", "Monster Name", "HP", "%","Crown");
```

Comment:
Let's always ensure that the total formatting line length it's the same - should be

Reply:
I increased the length of the displayed text fields so that we ensure a homogeneous line length. The following table summarize the changes:
Text Field | Flag | Old Text Format | New Text Format
---|:-:|:-:|:-:
Player Name | any | `%-32s`| `%-39s`
Monster Name | `-m` | `%-32s`| `%-39s`
Monster Name | `-m -c` | `%-32s`| `%-32s`

The pictures below show the two different layouts possible:

Execution of `sudo ./linux-hunter -m`
![image](https://user-images.githubusercontent.com/20427265/107264720-b64a9580-6a19-11eb-8e02-8106c895b8d0.png)

Execution of `sudo ./linux-hunter -m -c`
![image](https://user-images.githubusercontent.com/20427265/107264739-be0a3a00-6a19-11eb-8268-90763b03d4f4.png)

Reply:
Perfect - as long as there around enough spaces for all the crowns types (i.e. _bronze_, _silver_, _gold_) we should be ok.